### PR TITLE
docs(oracle): clarify slash window

### PR DESCRIPTION
**Pull request by adamewozniak**
## Description

Clarifies the slash window docs for validators

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/oracle/README.md [hunk 1]
```diff
@@ -81,7 +81,11 @@ A `VotePeriod` during which either of the following events occur is considered a
 
 - The validator fails to vote within the `reward band` around the weighted median for one or more denominations.
 
-During every `SlashWindow`, participating validators must maintain a valid vote rate of at least `MinValidPerWindow` (5%), lest they get their stake slashed (currently set to 0.01%). The slashed validator is automatically temporarily "jailed" by the protocol (to protect the funds of delegators), and the operator is expected to fix the discrepancy promptly to resume validator participation.
+A `SlashWindow` is a window of time during which validators can miss votes. At the end of this period, the amount of misses are tallied and the proper reward or punishment is carried out.
```
Comment (robert-zaremba) [inline x/oracle/README.md:84]:
```suggestion
A `SlashWindow` is a period duration when we count validators miss votes. At the end of every period, the amount of misses are tallied and the proper reward or punishment is carried out.
```

File: x/oracle/README.md [hunk 2]
```diff
@@ -81,7 +81,11 @@ A `VotePeriod` during which either of the following events occur is considered a
...
+
+During every `SlashWindow` (currently set to 7 days), participating validators must maintain a valid vote rate of at least `MinValidPerWindow` (5%), lest they get their stake slashed (currently set to 0.01%). The slashed validator is automatically temporarily "jailed" by the protocol (to protect the funds of delegators), and the operator is expected to fix the discrepancy promptly to resume validator participation.
```
Comment (robert-zaremba) [inline x/oracle/README.md:86]: let's describe what happens if a validator won't fix & unjail.

File: x/oracle/README.md [hunk 3]
```diff
@@ -81,7 +81,11 @@ A `VotePeriod` during which either of the following events occur is considered a
...
+
+Since the `MinValidPerWindow` is currently set to 5% validator must not miss more than 95% of votes in order to be safe from jailing.
```
Comment (robert-zaremba) [inline x/oracle/README.md:88]:
```suggestion
`MinValidPerWindow` is currently set to 5%. This means validator must not miss more than 95% of votes in order to be safe from jailing.
```

**Review by robert-zaremba — APPROVED** [review]
pre-approving. Left few suggestions

File: x/oracle/README.md (hunk 1 above)
Comment (adamewozniak) [reply x/oracle/README.md]: I think `window of time during which` is more clear - switched "this period" -> "every period"

**Pull request merged**